sysutils/cfiles/Portfile

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        mananapr cfiles 1.5 v
+categories          sysutils
+platforms           darwin
+license             MIT
+maintainers         {@sikmir gmail.com:sikmir} openmaintainer
+
+description         A ncurses file manager written in C with vim like keybindings
+long_description    ${name} is a terminal file manager with vim like keybindings, \
+                    written in C using the ncurses library. It aims to provide an \
+                    interface like ranger while being lightweight, fast and minimal.
+
+homepage            https://github.com/mananapr/cfiles
```

Comment:
The homepage is set to this by the GitHub portgroup, so this line should be removed because it is redundant.

Reply:
Fixed.